core/src/broadcast_stage/slot_transmit_shreds_cache.rs

```diff
@@ -0,0 +1,675 @@
+use super::*;
+use solana_ledger::shred::Shred;
+use solana_sdk::{clock::Slot, pubkey::Pubkey};
+use std::collections::VecDeque;
+
+pub type TransmitShreds = (Option<Arc<HashMap<Pubkey, u64>>>, Arc<Vec<Shred>>);
```

Comment:
comments about what this stuff is? why option

Reply:
Seems like HashMap<Pubkey, u64> can be a type and Arc<Vec<Shred>>.

Reply:
@sakridge `TransmitShreds` was introduced before this PR and is plumbed elsewhere through BroadcastStage, so I didn't refactor it.

I suspect it's an option because some tests may want to give turbine "Default" stakes?